### PR TITLE
NR-12569 :  header section in mobile and web view

### PR DESCRIPTION
**Pull request by dikshab0202**
JIRA : https://issues.newrelic.com/browse/NR-11585
subtask: https://issues.newrelic.com/browse/NR-12569
Description: 
1. Header is sticky in web view and non-sticky in mobile view
2. z-index=0 applied for header  bottom border line in mobile view
3. new relic logo size lesser in mobile view than web view
4. adjusted font-size and height
5. hover color is now same.

After change
image

Screenshot 2022-05-25 at 7 29 41 AM

Before change
image

image

### CODE REVIEW

**Comment by dikshab0202** [comment]
> Please sync this branch with the main branch. I have seen that the main branch got updated with a fix which I do not see in this branch. Thank you.

@pkosanam , okay , it is done.

**Comment by pkosanam** [comment]
Observed these things to be fixed:
1. On header, "login" and "Get started" should have styles on hover
2. Focus styles on the header are not similar to newrelic.com

**Comment by dikshab0202** [comment]
> Observed these things to be fixed:
> 
> 1. On header, "login" and "Get started" should have styles on hover
> 2. Focus styles on the header are not similar to newrelic.com

Hi @pkosanam , worked on above points, please review and let me know if any suggestions.

**Comment by pkosanam** [comment]
Observed things which has to be fixed
1. color of the menu options are not same as newrelic.com
2. Login and Get started are moving when hovered

3. Styles when focused(on links and also on the buttons on mobile view) are not matching as per newrelic.com
image

4. button on the carousel is hidden behind the card
image

File: src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js [hunk 1]
```diff
@@ -104,31 +104,42 @@ const GlobalHeader = ({ className, activeSite }) => {
...
-            height: var(--global-header-height);
             display: flex;
-            margin: 0 auto;
-            max-width: 1248px;
-            justify-content: space-between;
-            padding: 0 var(--site-content-padding);
-            height: 92px;
             align-items: center;
+            height: 5.75rem;
+            padding: 0 3.75rem;
```
Comment (pkosanam) [inline src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js:128]: I see that --site-content-padding has 3.75rem so you can use the below instead
padding: 0 var(--site-content-padding);

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Comment by pkosanam** [comment]
On 1127px, mobile menu is not displayed
image

**Comment by dikshab0202** [comment]
> On 1127px, mobile menu is not displayed image

Hi @pkosanam , it is fixed now.

File: src/components/styles.scss [hunk 2]
```diff
@@ -10,10 +10,12 @@
   --background-grey-color : #D6D6D6;
   --btn-background-green: #1CE783;
   --category-hover-color: #E4E5E6;
+  --black-hover-color: #1D252C;
   --site-content-padding: 3.75rem !important;
   @media (max-width: 960px) {
     --site-content-padding: 2.5rem !important;
   }
+  --global-header-height: 4rem !important;
```
Comment (pkosanam) [inline src/components/styles.scss:18]: cross check if height is changed on responsive screens

**Review by pkosanam — CHANGES_REQUESTED** [review]
application is working fine, please make these code changes

File: src/components/styles.scss (hunk 2 above)
Comment (dikshab0202) [reply src/components/styles.scss:18]: this height is only applicable for toggle screen( responsive) according to code structure.

File: src/pages/index.jsx [hunk 3]
```diff
@@ -213,26 +213,29 @@ const QuickstartsPage = ({ data, location }) => {
...
-                  css={css`
+      <RightArrowSVG
+        className="slick-next"
+        css={css`
                     width: auto;
                     height: auto;
                     margin: 0 1.5rem;
                   `}
-                />
-              </button>,
```
[the remark this answers is not in the document]
Comment (dikshab0202) [reply src/pages/index.jsx:235]: okay. done

File: src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js [hunk 4]
```diff
@@ -104,31 +104,42 @@ const GlobalHeader = ({ className, activeSite }) => {
...
+            justify-content: space-between;
+           
+            @media screen and (max-width: ${NAV_BREAKPOINT}) {
+              margin: 0 13.5rem; 
+            }
+
+            @media screen and (min-width: ${NAV_BREAKPOINT}) {
+              margin: 0 auto;
+              max-width: 1248px;
+              justify-content: space-between;
```
[the remark this answers is not in the document]
Comment (dikshab0202) [reply src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js]: okay. done.

File: src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js [hunk 5]
```diff
@@ -358,14 +426,20 @@ const GlobalHeader = ({ className, activeSite }) => {
...
               align-items: center;
-              margin-right: 3rem;
               height: 100%;
+              outline: none;
+             
+              @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
+                width: 7.5rem;
+              }
+              @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+                width: 10.625rem;
```
Comment (dikshab0202) [inline src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js:436]: This is the logo width size breakpoint.

File: src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js [hunk 6]
```diff
@@ -338,12 +401,17 @@ const GlobalHeader = ({ className, activeSite }) => {
...
             width: 100%;
-            padding: 1.5rem;
             flex-wrap: wrap;
             justify-content: space-between;
             flex-direction: column;
-            @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
+            @media screen and (max-width: ${NAV_BREAKPOINT}) {
               flex-direction: row;
+              padding: 1rem;
+            }
```
Comment (dikshab0202) [inline src/@newrelic/gatsby-theme-newrelic/components/GlobalHeader.js:410]: This one is the breakpoint when we can see toggle - 1127px

**Pull request merged**